src/components/HearingList.js

```diff
@@ -57,16 +103,30 @@ HearingListItem.contextTypes = {
 
 class HearingList extends React.Component {
   render() {
-    const {state, data} = (this.props.hearings || {});
-    if (state !== "done") return <LoadSpinner />;
-    return (<div className="hearing-list">{data.map(
-      (hearing) => <HearingListItem hearing={hearing} key={hearing.id}/>
-    )}</div>);
+    const {hearings, isLoading, labels, handleChangeFilter, handleSort, handleSearch} = this.props;
+
+    return (
+      <div>
+        {isLoading && <LoadSpinner />}
```

Comment:
I would move the LoadSpinner below the tabs, so that the search boxes are visible all the time. Only the list would be updated (displaying the spinner) when something is typed in the search boxes.